scripts/travis/integration_test.sh

```diff
@@ -15,16 +15,6 @@ export ALGORAND_DEADLOCK
 SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
 
 export BUILD_TYPE="integration"
-
-
-if [ "${USER}" = "travis" ]; then
-    # we're running on a travis machine
-    "${SCRIPTPATH}/travis_wait.sh" 120 "${SCRIPTPATH}/build.sh" --make_debug
-    "${SCRIPTPATH}/travis_wait.sh" 120 "${SCRIPTPATH}/test.sh"
-else
-    # we're running on an ephermal build machine
-    "${SCRIPTPATH}/build.sh" --make_debug
-    "${SCRIPTPATH}/test.sh"
-fi
+"${SCRIPTPATH}/build_test.sh"
```

Comment:
I agree that we need to "untangle" these scripts. Maybe try to draw the call stack for all of these from the top-bottom ?

Reply:
Good idea, I created a ticket for it and can look at that soon: https://github.com/algorand/go-algorand-internal/issues/1291